### DISCUSSION TITLE
最近在用univer3.0版本，遇到几个问题想请教一下

### DISCUSSION THREAD

**Discussion by laojun** [dream-num/univer#3674] [Q&A | 常见问题]
1、mergeMenu的时候，添加如下，但是无效
![image](https://github.com/user-attachments/assets/ac35eaec-0bac-48c0-b3a1-118415d0b7d1)

```
[ContextMenuPosition.MAIN_AREA]: {
        [ContextMenuGroup.OTHERS]: {
          [SingleButtonOperation.id]: {
            order: 0,
            menuItemFactory: CustomMenuItemSingleButtonFactory,
          },
    },
}
```
2、通过右键自定义菜单给单元格添加custom配置后，如何根据custom配置在单元格中增加不同标记样式：比如custom: {editable: false},需要给单元格添加一个disabeld的样式

3、在定义菜单动作的handler中能否处理单元格信息？比如点击按钮后，对获焦区域每个单元格进行样式和值的设置
![image](https://github.com/user-attachments/assets/132b6293-ade8-419d-b8c1-94024129777d)
![image](https://github.com/user-attachments/assets/418ff436-95d4-4f49-808e-441f00966648)

十分感谢

**Comment by 13701138894** [comment]
![image](https://github.com/user-attachments/assets/b245dbfb-9e02-46d5-93a4-5fc816b14601)
换成其他的试试，如FORMAT

**Comment by 13701138894** [comment]
第三个问题我也在问 #3685
